### DISCUSSION TITLE
Bootstrap doesn't work anymore in windows 7?

### DISCUSSION THREAD

**Discussion by jeyvix** [twbs/bootstrap#36512] [Q&A] [answered]
We have a site with bootstrap 5 code and after last updates of chrome and edge they don't upload css anymore, the only other browser that still works is mozilla firefox. We are using windows 7, while on win 10 and android it works. If I delete history navigation it works for one only upload and then when I refresh it doesn't work again.
Is there someone who have the same problem? What's the problem: windows 7, bootstrap or browsers?

**Comment by mdo** [comment, marked as answer]
You can read about our browser support here: https://getbootstrap.com/docs/5.2/getting-started/browsers-devices/. We don't target specific versions of Windows—we target by browser. Bootstrap 5 dropped support for IE, but not Microsoft Edge. No changes have been made to browser support since the initial v5.0.0 release.

**Reply by jeyvix** [reply]
Thanks for the answer! So you're saying that the alteration is done by browsers? We have problem with both edge and chrome from their last update. Probably is there a code that they didn't considerated for keeping css of bootstrap?

**Reply by mdo** [reply]
I can’t see them dropping support for something that would cause huge issues for you. Perhaps you’re running into a mismatched integrity SHA on the `<link>` element when including Bootstrap? Or something else is at play maybe? There’s been no change to supported browsers since v5 launched, so it must be something else I’d think
